js/classes/question.js

```diff
@@ -1,18 +1,25 @@
-class Questions extends Game {
-  constructor(round, gameState, categoryId, question, answer, pointValue) {
-    super(round, gameState);
+class Question extends Game {
```

Comment:
Question also shouldn't be extending Game --- `extends` is specifically for the purpose of inheritance. You should both take some time to review that concept and pair with some other students to gain insight into their understanding of when you do/don't want to inherit.